doc/parallel.rst

```diff
@@ -39,22 +39,22 @@ This backend will use MPI (Message Passing Interface) on the system to split neu
 
 **MacOS Environment**::
 
-    $ export LD_LIBRARY_PATH=${CONDA_PREFIX}/lib
+    $ export DYLD_FALLBACK_LIBRARY_PATH=${CONDA_PREFIX}/lib
 
 Alternatively, run the commands below will avoid needing to run the export command every time a new shell is opened::
 
     $ cd ${CONDA_PREFIX}
     $ mkdir -p etc/conda/activate.d etc/conda/deactivate.d
-    $ echo "export OLD_LD_LIBRARY_PATH=\$LD_LIBRARY_PATH" >> etc/conda/activate.d/env_vars.sh
-    $ echo "export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:\${CONDA_PREFIX}/lib" >> etc/conda/activate.d/env_vars.sh
-    $ echo "export LD_LIBRARY_PATH=\$OLD_LD_LIBRARY_PATH" >> etc/conda/deactivate.d/env_vars.sh
-    $ echo "unset OLD_LD_LIBRARY_PATH" >> etc/conda/deactivate.d/env_vars.sh
+    $ echo "export OLD_DYLD_FALLBACK_LIBRARY_PATH=\$DYLD_FALLBACK_LIBRARY_PATH" >> etc/conda/activate.d/env_vars.sh
+    $ echo "export DYLD_FALLBACK_LIBRARY_PATH=\$DYLD_FALLBACK_LIBRARY_PATH:\${CONDA_PREFIX}/lib" >> etc/conda/activate.d/env_vars.sh
+    $ echo "export DYLD_FALLBACK_LIBRARY_PATH=\$OLD_DYLD_FALLBACK_LIBRARY_PATH" >> etc/conda/deactivate.d/env_vars.sh
+    $ echo "unset OLD_DYLD_FALLBACK_LIBRARY_PATH" >> etc/conda/deactivate.d/env_vars.sh
 
```

Comment:
I'm wondering if this is too much magic and subject to failure. I normally just add environment variables to bashrc or bash_profile.

Reply:
I was considering that too, though I've personally found the technique here to be very dependable. Problems only emerge when the OS changes its use of certain obscure environmental variables like `LD_LIBRARY_PATH`. I'm guessing @blakecaldwell came up with this to avoid spaghettifying peoples' environmental variables via bashrc/bash_profile as has been done in the past. It's also important to note that the current issue solved by this PR would have also emerged if we used bashrc/bash_profile.

Reply:
For sure. Are you in a rush to merge this? Otherwise I can test this out tomorrow. Set it to [MRG] when ready from your end, so I know I can merge when I've tested it out.

Reply:
The one for HNN-GUI needs to be done soon for the workshop participants, but I think this one for HNN-core is less time sensitive.

Reply:
So I tested this out on an M1 mac ... somehow the OpenMPI install itself wasn't working. Also noticed that the you're missing a leading forward slash in front of the paths. They should be /etc/.../ not etc/.../

Reply:
Hmm, that's weird. What went wrong with the OpenMPI installation?

Using `/etc` vs `etc` will put this at root instead of the `$CONDA_PREFIX` we've cd'd into.

Reply:
ahhh ... I just tried:

```sh
conda install -c conda-forge openmpi
```

but then `mpiexec` could not be found in the path. If I tried the command on your documentation, it ends up giving unsatisfiable requirements error.

Reply:
Which command? That doesn't sound like an error stemming from the content in this PR....

Reply:
sure ... I just wasn't able to test this PR because install Open-MPI doesn't work :) We can merge and move on if you like

Reply:
That's annoying for sure though. I wish we could've tested it. I wonder what Chris did to get MPI installed....